scripts/base/init-bare.zeek

```diff
@@ -213,11 +213,12 @@ type transport_proto: enum {
 ##    see :ref:`the manual's description of the connection record
 ##    <writing-scripts-connection-record>`.
 type conn_id: record {
-	orig_h: addr;	##< The originator's IP address.
-	orig_p: port;	##< The originator's port number.
-	resp_h: addr;	##< The responder's IP address.
-	resp_p: port;	##< The responder's port number.
-} &log;
+	orig_h: addr &log;	##< The originator's IP address.
+	orig_p: port &log;	##< The originator's port number.
+	resp_h: addr &log;	##< The responder's IP address.
+	resp_p: port &log;	##< The responder's port number.
+	proto: count &default=65535;  ##< The transport protocol ID. Defaults to 65535 as an "unknown" value.
```

Comment:
I'm tempted to add a script-layer `UNKNOWN_IP_PROTO` constant to mirror the core, but I think it doesn't matter much — the default will mean that users generally won't use the value explicitly. But it would help avoid proliferating the literal. Feel free to handle as you see fit.

Reply:
I like it!

Reply:
I'm actually going to leave this one out. It results in some cascading `#include` issues in C++ (Conn.h requires const.bif.netvar_h which requires Val.h...) which end up being more changes than it's worth.